packages/react/src/components/accordion/accordion-types.ts

```diff
@@ -0,0 +1,31 @@
+import { ReactNode, RefObject, KeyboardEvent } from 'react';
+import { Type, Tag } from '../heading/heading';
+
+export interface AccordionProps {
+    /** Unique id per accordion */
+    id: string;
+    children: ReactNode;
+    /** Multipe or single panel open concurrently */
+    mode?: 'single' | 'multi';
+}
+
+export interface AccordionItemProps {
+    /** Title Label */
+    title: string;
+    /** Unique Id on each accordion per group */
+    id?: string;
+    /** This is relate to styling */
```

Comment:
This one too

Reply:
removed the comments that was being used on storybook